### DISCUSSION TITLE
Metagroups

### DISCUSSION THREAD

**Discussion by 2q2code** [creativeprojects/resticprofile#438] [Ideas]
Groups are cool, but you know what would be cooler? Groups of groups!

That way I can schedule just one backup operation to run all the backups for a system sequentially, at a given time or times.

**Comment by creativeprojects** [comment]
What would it solve that you cannot do with groups?

**Reply by 2q2code** [reply]
Okay. I have static and dynamic profiles that are created based on databases in existence. So, for a single server, if I want to set up a schedule that backs up all profiles sequentially (ie not being overly thrashing or slowing down the network by overlapping) at a lull time in the early hours, I would prefer to schedule only ONE operation instead of guessing how long each one will take and have to set up a schedule for each group. I am using inheritance and includes, but merging for includes clobbers the content of yaml lists instead merging them. So I can't have all profiles in one group, as they are part of an included (and dynamic profile) file. I might need to show you config to explain because my reasoning here doesn't sound clear to me.

**Reply by creativeprojects** [reply]
If the problem is about merging lists, there's already a feature doing exactly that:

https://creativeprojects.github.io/resticprofile/configuration/inheritance/index.html#prepend--append-to-list-properties

**Reply by 2q2code** [reply]
Assuming the merging lists is fixed, how would you configure and enable scheduling for all profiles on a server in one pass? I attempted to follow the discussion on the V2 proposal, but `cannot load profile '*': not found`

**Reply by 2q2code** [reply]
Okay, so in my case, I am not sure if the inheritance link above applies, as it is in the groups section, which doesn't have explicit inheritance. I have for now split my database backups into _static and _dynamic, and that's fine. What was happening was that the database backup group for the statically specified database profiles was being obliterated when the dynamic database group was loaded by FILE inclusion (include directory of config files).
